framework/src/actions/ActionWarehouse.C

```diff
@@ -180,6 +180,15 @@ ActionWarehouse::getActionsByName(const std::string & task) const
   return it->second;
 }
 
+bool
+ActionWarehouse::hasActions(const std::string & task) const
+{
+  if (_action_blocks.find(task) == _action_blocks.end())
```

Comment:
You can simplify this to `return _action_blocks.find(task) != _action_blocks.end();`

This patch is fine though